File: algorithms/cpp/633.cpp
#include <iostream>
#include <cmath>
using namespace std;

class Solution {
private:
    bool isSquare(int n) {
        return pow((int)sqrt(n), 2) == n;
    }
public:
    bool judgeSquareSum(int c) {
        for (int a = 0; a <= sqrt(c); a++)
            if (isSquare(c-a*a))
                return true;
        return false;
    }
};

int main() {
    return 0;
}

File: algorithms/python/633.py
class Solution:
    def judgeSquareSum(self, c):
        """
        :type c: int
        :rtype: bool
        """
        for i in range(0, int(c**0.5)+1):
            a = i
            b = round((c - a**2) ** 0.5)
            if a**2 + b**2 == c:
                return True
        return False
